app/assets/javascripts/application.js

```diff
@@ -61,5 +61,4 @@
 //= require surveyor/surveys
 //= require sweetalert.min
 //= require update_visit_group
-//= require update_visit
```

Comment:
Removed most of the JS from update_visit.coffee, moved the click listener to service_calendar.js.coffee